realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -2302,7 +2302,11 @@ public void run() {
         assertTrue(Realm.deleteRealm(configuration));
 
         // Directory should be empty now.
```

Comment:
That note doesn't seem correct given that the length is 0. I assume it is the `.lock` file. Clarify this or just move the comment below up instead of this one?